src/ui/components/RatingManagerNotice/index.js

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+import makeClassName from 'classnames';
+import * as React from 'react';
+
+import Notice from 'ui/components/Notice';
+import type { NoticeType } from 'ui/components/Notice';
+
+import './styles.scss';
+
+type Props = {|
+  className?: string | null,
+  hideMessage: boolean,
+  message: string,
+  type: NoticeType,
```

Comment:
If the caller omits `type` (or something like that), you could render it without `Notice` and use this component for the existing `<span>` notification. What do you think? It seems to work except I didn't add in the CSS transition.

<details>
<summary>Example patch</summary>

```diff
diff --git a/src/amo/components/AddonReviewManager/index.js b/src/amo/components/AddonReviewManager/index.js
index 4ec38f176..b9e8f45c5 100644
--- a/src/amo/components/AddonReviewManager/index.js
+++ b/src/amo/components/AddonReviewManager/index.js
@@ -15,6 +15,7 @@ import translate from 'core/i18n/translate';
 import { sanitizeHTML } from 'core/utils';
 import DismissibleTextForm from 'ui/components/DismissibleTextForm';
 import Rating from 'ui/components/Rating';
+import RatingManagerNotice from 'ui/components/RatingManagerNotice';
 import type { AppState } from 'amo/store';
 import type { DispatchFunc } from 'core/types/redux';
 import type { ErrorHandlerType } from 'core/errorHandler';
@@ -98,17 +99,18 @@ export class AddonReviewManagerBase extends React.Component<InternalProps> {
             styleSize="small"
             yellowStars
           />
-          <span
-            className={makeClassName('AddonReviewManager-savedRating', {
-              'AddonReviewManager-savedRating-hidden':
-                flashMessage !== STARTED_SAVE_RATING &&
-                flashMessage !== SAVED_RATING,
-            })}
-          >
-            {flashMessage === STARTED_SAVE_RATING
-              ? i18n.gettext('Saving')
-              : i18n.gettext('Saved')}
-          </span>
+          <RatingManagerNotice
+            className="AddonReviewManager-savedRating"
+            hideMessage={
+              flashMessage !== STARTED_SAVE_RATING &&
+              flashMessage !== SAVED_RATING
+            }
+            message={
+              flashMessage === STARTED_SAVE_RATING
+                ? i18n.gettext('Saving')
+                : i18n.gettext('Saved')
+            }
+          />
         </div>
         <DismissibleTextForm
           formFooter={formFooter}
diff --git a/src/ui/components/RatingManagerNotice/index.js b/src/ui/components/RatingManagerNotice/index.js
index b3bc404ec..4bdccef56 100644
--- a/src/ui/components/RatingManagerNotice/index.js
+++ b/src/ui/components/RatingManagerNotice/index.js
@@ -11,7 +11,7 @@ type Props = {|
   className?: string | null,
   hideMessage: boolean,
   message: string,
-  type: NoticeType,
+  type?: NoticeType,
 |};
 
 const RatingManagerNotice = ({
@@ -20,16 +20,21 @@ const RatingManagerNotice = ({
   message,
   type,
 }: Props) => {
-  return (
-    <Notice
-      type={type}
-      className={makeClassName('RatingManagerNotice-savedRating', className, {
-        'RatingManagerNotice-savedRating-hidden': hideMessage,
-      })}
-    >
-      {message}
-    </Notice>
-  );
+  const props = {
+    className: makeClassName('RatingManagerNotice-savedRating', className, {
+      'RatingManagerNotice-savedRating-hidden': hideMessage,
+    }),
+  };
+
+  if (type) {
+    return (
+      <Notice type={type} {...props}>
+        {message}
+      </Notice>
+    );
+  }
+
+  return <span {...props}>{message}</span>;
 };
 
 export default RatingManagerNotice;
```

</details>